code/game/objects/items/weapons/lighters.dm

```diff
@@ -5,8 +5,8 @@
 	icon = 'icons/obj/items.dmi'
 	lefthand_file = 'icons/mob/inhands/lighter_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/lighter_righthand.dmi'
-	icon_state = "lighter-g"
-	item_state = "lighter-g"
+	icon_state = "lighter-"
```

Comment:
This means they will be invisible items when mapped in, which isnt ideal. Can you keep the icon_states as they are, and use `base_icon_state` for update_icon_state() and update_overlays()?